tests/parsers/beckman_vi_cell_blu/vi_cell_blu_parser_test.py

```diff
@@ -38,14 +38,13 @@ def test_get_model() -> None:
     parser = ViCellBluParser(TimestampParser())
     result = parser._get_model(get_data(), get_filename())
 
-    cell_counting_document_item = (
-        result.cell_counting_aggregate_document.cell_counting_document[0]
+    cell_counting_document = (
```

Comment:
```suggestion
    assert result.cell_counting_aggregate_document
    cell_counting_document = result.cell_counting_aggregate_document.cell_counting_document
```

You can assert on that the agg document is not None to remove the typing issue